src/drivers/imx/sdma.c

```diff
@@ -92,7 +92,7 @@ static void sdma_set_overrides(struct dma_chan_data *channel,
 
 static void sdma_enable_channel(struct dma *dma, int channel)
 {
-	dma_reg_write(dma, SDMA_HSTART, BIT(channel));
+	dma_reg_update_bits(dma, SDMA_HSTART, BIT(channel), BIT(channel));
```

Comment:
Does this actually have an effect? From what I recall from documentation, writing a 0 does NOT clear the bit but rather it leaves it unchanged, and writing a 1 does the actual action of setting the actual internal start bit. I think the old code is the correct one and this one could accidentally start other channels more times than required.